src/go/configgenerator/cluster_generator.go

```diff
@@ -205,6 +225,26 @@ func makeBackendCluster(opt *options.ConfigGeneratorOptions, brc *sc.BackendRout
 		LoadAssignment:       util.CreateLoadAssignment(brc.Hostname, brc.Port),
 	}
 
+	if opt.DnsResolverAddress != "" {
+		host, port, err := parseDnsResolverAddress(opt.DnsResolverAddress)
+		if err != nil {
+			return nil, fmt.Errorf("fail to parse dnsResolverAddress: %v", err)
+		}
+
+		c.DnsResolvers = []*corepb.Address{
```

Comment:
We need to add dns_resovler for all clusters;  service control,  jwks_url,  service_management, iam

Reply:
add to all clusters